src/components/MUITypography/Typography.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { createMuiTheme, ThemeProvider, Typography } from '@material-ui/core';
+import { select, text } from '@storybook/addon-knobs';
+import React from 'react';
+import { ProgressButton } from '../ProgressButton';
+
+export default { title: 'Components/Typography' };
+
+const theme = createMuiTheme({
+  typography: {
+    fontFamily: 'Arial',
+    button: {
+      fontSize: 10,
+    },
+  },
+});
+
+export const Story = () => (
+  <ThemeProvider theme={theme}>
+    <Typography
+      variant={select(
+        'Variant',
+        ['body1', 'caption', 'h1', 'h5', 'subtitle1'],
+        'body1'
+      )}
+      color={select(
+        'Color',
+        ['primary', 'secondary', 'error', 'textPrimary'],
+        'secondary'
```

Comment:
```suggestion
        ['initial', 'inherit', 'primary', 'secondary', 'error', 'textPrimary', 'textSecondary'],
        'initial'
```

Reply:
Actually, I was gonna add special options. but i will keep to add all options for future stories. thanks